components/tekton-ci/production/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ resources:
 - ../base
 - ../base/external-secrets
 - plnsvc-ci-secret.yaml
+- plnsvc-codecov-secret.yaml
```

Comment:
@Roming22 it should be `plnsvc-codecov.yaml`

Reply:
You better rename plnsvc-codecov.yaml to plnsvc-codecov-secret.yaml

Reply:
Fixed